packages/wrangler/src/__tests__/kv.test.ts

```diff
@@ -212,7 +212,7 @@ describe("wrangler", () => {
 
       it("should make multiple requests for paginated results", async () => {
         // Create a lot of mock namespaces, so that the fetch requests will be paginated
-        const KVNamespaces = [];
+        const KVNamespaces: { title: string; id: string }[] = [];
```

Comment:
Should this be a named type?

Reply:
I think it would clarify the API if it was